src/components/Question.js

```diff
@@ -27,10 +27,20 @@ export default function Question({
 }) {
   const [showReply, setShowReply] = useState(false)
   const [firstReply] = replies
+  const [resolvedBy, setResolvedBy] = useState(question?.resolved_reply_id)
+  const [resolved, setResolved] = useState(question?.resolved)
   const questionAuthorId = firstReply?.profile?.id
+  const handleResolve = async (resolved, replyId = null) => {
+    await supabase
+      .from('squeak_messages')
+      .update({ resolved, resolved_reply_id: replyId })
+      .match({ id: question?.id })
```

Comment:
I think we should use the API for this, so we can ditch supabase easier in the future (/api/question/resolve)